modules/otel-forwarding-logs-to-tempostack.adoc

```diff
@@ -6,7 +6,8 @@
 [id="otel-forwarding-logs-to-tempostack_{context}"]
 = Forwarding logs to a LokiStack instance
 
-You can deploy the OpenTelemetry Collector to forward logs to a LokiStack instance.
+You can deploy the OpenTelemetry Collector to forward logs to a LokiStack instance with `openshift-logging` tenants mode.
+For more information about the supported LokiStack configuration refer to the logging documentation.
```

Comment:
@max-cx could you please add a link to the logging docs https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/logging/index ?